tests/test_meshio.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+import os
+
+import numpy
+
+import pyvista
+
+from pyvista import examples
+
+
+def test_beam():
+    # Load reference mesh
+    mesh_ref = pyvista.UnstructuredGrid(examples.hexbeamfile)
+
+    # Save and read reference mesh using meshio
+    pyvista.save_meshio("test_meshio.vtk", mesh_ref)
+    mesh = pyvista.read_meshio("test_meshio.vtk")
+    os.remove("test_meshio.vtk")
```

Comment:
Can you use `tmpdir` with PyTest much like other tests: https://github.com/pyvista/pyvista/blob/60c96c3903deff4acde2f3f19996fcc8f5511a26/tests/test_polydata.py#L256